.travis.yml

```diff
@@ -1,7 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-   - "4"                    # Maintenance LTS release
+   - '6'
+   - '8'
+   - '10'
```

Comment:
Isn't 10 covered by `lts/*` if `lts/*` means "active LTS release"? Or is it that we want to hard-code 10 because we want to keep tests running when 10 becomes not active?

Reply:
The idea is to ensure Node10 always gets tested even when it is not LTS.

Reply:
I would vote to either pin the versions or use lts/* always - I think we historically pinned since dropping support for LTS meant (usually) bumping major version, so someone always had to do something to make it happen (vs CI just automatically dropping tests for that version).

Reply:
We are using `lts/*` which means it will cover active LTS but not maintenance LTS. For maintenance LTS you need to specify the specific versions. (Atleast that is what my understanding is)